syntax = "proto2";
package protos;

message HiveConfig {
  // hive master's ip
  required string host = 1;

  // hive port
  required uint32 port = 2 [default = 10000];

  // hive username
  required string username = 3 [default = 'admin'];

  // hive database
  required string database = 4 [default = 'default'];

  required string table_name = 5;
}
